set.mm

```diff
@@ -419688,16 +419772,58 @@ the sequence p(0) p(1) ... p(n-1) of the vertices in a closed walk p(0)
       VDRUIVIVNVOVKVIVNVOVGVNVOQVHVGVAVNVEVGVAVNVEQVGVAOZVNVEVPVMVCVDVPVLVBVBVF
       BUJUKULUMUNUOUPTUQURT $.
 
-    $( A closed walk of length 1 is a loop.  (Contributed by AV,
-       24-Apr-2021.) $)
-    clwwlksn1loop $p |- ( W e. ( 1 ClWWalksN G ) -> ( ( # ` W ) = 1
-                             /\ { ( W ` 0 ) , ( W ` 0 ) } e. ( Edg ` G ) ) ) $=
-      ( vi c1 cclwwlksn co wcel cvtx cfv cword chash wceq wa caddc cpr cedg cc0
-      cv cmin eqid cfzo wral clsw w3a clwwlknp adantr lsw1 preq1d eleq1d biimpa
-      simpr jca 3adant2 syl ) BDAEFGBAHIZJGZBKIDLZMZCRZBIUSDNFBIOAPIZGCQDDSFUAF
-      UBZBUCIZQBIZOZUTGZUDUQVCVCOZUTGZMZCUTADUOBUOTUTTUEURVEVHVAURVEMUQVGURUQVE
-      UPUQUKUFURVEVGURVDVFUTURVBVCVCUOBUGUHUIUJULUMUN $.
+    $( A closed walk of length 1 represented as word is a word consisting of 1
+       symbol representing a vertex connected by (at least) one edge with
+       itsself, i.e., by a loop.  (Contributed by AV, 24-Apr-2021.)  (Revised
```

Comment:
itself
(and maybe: "connected to itself a (at least) one edge, that is, a loop")

Reply:
Done